pkg/eventservice/event_service.go

```diff
@@ -54,6 +54,7 @@ type DispatcherInfo interface {
 	GetBdrMode() bool
 	GetIntegrity() *integrity.Config
 	GetTimezone() *time.Location
+	GetIsRedo() bool
```

Comment:
IsRedo() is better.

Reply:
It is generated by `eventpb.DispatcherRequest`.